### PR TITLE
Miscellaneous improvements.

### PR DESCRIPTION
**Pull request by ejball**
Review commit by commit for rationale. We can pick and choose if we want.

### CODE REVIEW

File: tests/Directory.Build.props [hunk 1]
```diff
@@ -6,4 +6,8 @@
     <IsPackable>false</IsPackable>
   </PropertyGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Faithlife.Analyzers" Version="1.0.5" />
```
Comment (bgrainger) [inline tests/Directory.Build.props]: `PrivateAssets="all"`? (Here and in `src/`, too.)

File: src/ProjectName/ProjectName.csproj [hunk 2]
```diff
@@ -1,7 +1,6 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <VersionPrefix>0.1.0</VersionPrefix>
```
Comment (ejball) [inline src/ProjectName/ProjectName.csproj]: Thinking about reverting this. Too confusing.

**Comment by ejball** [comment]
Any objection to merging, @bgrainger, @ddunkin?

File: src/Directory.Build.props [hunk 3]
```diff
@@ -19,10 +17,12 @@
     <PublishRepositoryUrl>true</PublishRepositoryUrl>
     <EmbedUntrackedSources>true</EmbedUntrackedSources>
     <AllowedOutputExtensionsInPackageBuildOutputFolder>$(AllowedOutputExtensionsInPackageBuildOutputFolder);.pdb</AllowedOutputExtensionsInPackageBuildOutputFolder>
+    <IsPackable>false</IsPackable>
```
Comment (ddunkin) [inline src/Directory.Build.props]: What's the rationale for this?

Comment (ejball) [reply src/Directory.Build.props]: Nervousness about deploying a NuGet package for a project just because it is placed in `src`.

Comment (ddunkin) [reply src/Directory.Build.props]: How often do we have a project in `src` that shouldn't be packaged? (Web projects are not packaged by default.)

What happens if ProjectA is packable and references ProjectB which is not packable? Will it fail, or will it create a ProjectA package with a reference to a non-existent ProjectB?

Comment (ejball) [reply src/Directory.Build.props]: Good questions. Perhaps I'm being unnecessarily paranoid.

**Review by bgrainger — APPROVED** [review]

File: src/Directory.Build.props (hunk 3 above)
Comment (ejball) [reply src/Directory.Build.props]: Okay, dropped the unnecessary references to `IsPackable` and `IsTestProject`. @ddunkin, do you approve?

**Review by ddunkin — APPROVED** [review]

**Pull request merged**